doc/tutorials/sagemaker/sme_init.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>Did we hard code the alias in the script deployer?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yes, I think it is hardcoded.</p>